chain/src/chain.rs

```diff
@@ -43,13 +43,14 @@ impl Drop for ChainController {
 }
 
 impl ChainController {
-    pub fn process_block(&self, block: Arc<Block>) -> Result<(), FailureError> {
-        Request::call(&self.process_block_sender, block).expect("process_block() failed")
+    pub fn process_block(&self, block: Arc<Block>, need_verify: bool) -> Result<(), FailureError> {
+        Request::call(&self.process_block_sender, (block, need_verify))
+            .expect("process_block() failed")
```

Comment:
Suggest to add another test-used-only interface like `process_block_without_verifying`.